src/lib/Scenes/Fair/Components/FairHeader/CountdownTimer.tsx

```diff
@@ -14,33 +15,22 @@ enum FairTimerState {
   PAST = "PAST",
 }
 
-export const formatDate = date => {
-  const momentToUse = moment(date)
-  const momentDate = momentToUse.format("MMM D")
-  const momentHour = momentToUse.format("ha")
-  if (!!momentHour && !!momentDate) {
-    return ` ${momentDate} at ${momentHour}`
-  } else if (!!momentDate) {
-    return ` ${momentDate}`
-  }
-}
-
-function relevantStateData(state, { startAt, endAt }: Props) {
+function relevantStateData(state, { startAt, endAt, formattedOpeningHours }: Props) {
   switch (state) {
     case FairTimerState.UPCOMING:
       return {
         date: startAt,
-        label: `Opens${formatDate(startAt)}`,
+        label: formattedOpeningHours,
       }
     case FairTimerState.CURRENT:
       return {
         date: endAt,
-        label: `Closes${formatDate(endAt)}`,
+        label: formattedOpeningHours,
       }
     case FairTimerState.PAST:
       return {
         date: null,
-        label: "Closed",
+        label: formattedOpeningHours,
       }
   }
```

Comment:
This block could probably be DRY-ed up a bit, but no blocker.

Reply:
Yeah I at first refactored this, but in the auction timer and this component share a lot of code and `label` is used as the display, so I wan't to keep things consistent and not disrupt the status quo.